component/_buttons.scss

```diff
@@ -1,57 +1,82 @@
 // @file buttons.scss
 
-$form-button-background: palette(brand);
-$form-button-border: false;
-$form-button-border-color: palette(brand, dark);
+// Remove and edit styles as needed. This basic idea provides three button styles
+// Can easily be expanded to accomodate more
+
+$form-button-background-default: palette(mono, 10);
+$form-button-color-default: palette(mono, 66);
+$form-button-border-color-default: palette(mono, 33);
 $form-button-border-width: 1px;
 $form-button-border-radius: $default-border-radius;
-$form-button-padding: .325em 1em .25em;
-
-// Default button style. Classes must be chained to change colors, i.e. class="button button--primary"
-a.button,
-button.button,
-input.button {
-	@include appearance(button);
-	@include transition( all 250ms ease );
-	border-radius: $form-button-border-radius;
-	cursor: pointer;
-	margin-bottom: .5em;
-	padding: $form-button-padding;
+$form-button-padding: .625em 1em .5em;
+
+//
+// Usage: @include button()
+// Accepts: null (default), primary, secondary
+// Returns: A compiled set of buttons styles including idle and hover
+// Authors: The Oomph Team
+//
+@mixin button($type:default) {
+  border: $form-button-border-width solid $form-button-border-color-default;
+  border-radius: $form-button-border-radius;
+  display: inline-block;
+  padding: $form-button-padding;
+  text-transform: uppercase;
+
```

Comment:
Am I way off in saying that if we are only using these vars for these buttons then we truly dont need them. The initial `button()` mixin is in fact setting the defaults for each child button so its inherently inherited! Just a thought.